src/modules/auth/auth-utils.ts

```diff
@@ -41,14 +41,14 @@ export const authSignInWithPhoneNumber = async (
     await authenticateWithSms(phoneNumberWithPrefix, language);
     dispatch({type: 'SIGN_IN_INITIATED', phoneNumber: phoneNumberWithPrefix});
   } catch (e: any) {
-    const error = getAxiosErrorMetadata(e);
-    if (error.responseStatus === 400) {
-      const errorData = error.responseData && JSON.parse(error.responseData);
+    const error = e as HttpErrorResponse;
```

Comment:
We can't cast to `HttpErrorResponse`, right? Since the `http` field might be undefined.